bin/clang/pragmas.cc

```diff
@@ -481,13 +474,21 @@ SSTAssumeTruePragma::activate(Stmt *s)
 {
   IfStmt* ifs = cast<IfStmt>(s);
```

Comment:
Do we know that this is an IfStmt with certainty?

Reply:
added check